cmd/scaffold.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/onflow/flow-go/module/metrics"
 	"github.com/onflow/flow-go/module/trace"
 	"github.com/onflow/flow-go/network"
-	jsoncodec "github.com/onflow/flow-go/network/codec/json"
+	cborcodec "github.com/onflow/flow-go/network/codec/cbor"
```

Comment:
This is a great optimization, but maybe make this change in a separate PR, keeping the concern of this PR to add the binstat tool

Reply:
@turbolent thanks for the comment! Okay, so I pushed a change whereby `cmd/scaffold.go` still uses JSON instead of CBOR. So this PR is basically the `binstat` package and tests + some `binstat` instrumentation of `flow-go` sources + unused CBOR codec with tests. Once this PR goes through I can then make a 2 line PR to switch the networking to use CBOR instead of JSON...

Why didn't I just rip out all the CBOR stuff from this PR? I try to be lazy in a good way :-)